oat-sa/tao-core
# Fix/ncc-126/translate select2

**Related to task:**  https://oat-sa.atlassian.net/browse/NCC-126
**Description:** [solution for adding translations to select2 library](https://oat-sa.atlassian.net/wiki/spaces/~105352349/pages/237797807/Translation+with+select2)
**Requires:** 

- [x] [PR](https://github.com/oat-sa/tao-core-libs-fe/pull/16) to `tao-core-libs` is merged
- [x] new npm `tao-core-libs` version is released 
- [x] Update `tao-core-libs` version in package.*json files
- [x] Update [docs](https://oat-sa.atlassian.net/wiki/spaces/~105352349/pages/237797807/Translation+with+select2) and move to [the proper space](https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/4227119/Frontend)
**How to test:** 
- Create new or apply changes to the tao instance 
- Navigate to the "My Settings" page and change "Interface language" to any available (For example Japanese)
- Navigate to the "Deliveries" page and click on "New delivery"
- Messages in the search field should be translated

![Screenshot 2020-02-26 at 16 56 14](https://user-images.githubusercontent.com/6339002/75362571-39321900-58b9-11ea-8849-8b0e1e098fb0.png)
